cmd/frontend/graphqlbackend/external_services_test.go

```diff
@@ -114,6 +122,49 @@ func TestAddExternalService(t *testing.T) {
 				t.Fatalf("NamespaceUserID: want %d but got %d", userID, *result.externalService.NamespaceUserID)
 			}
 		})
+
+		t.Run("user mode disabled but user has public tag", func(t *testing.T) {
```

Comment:
nit: just to align with the tone with other tests :D
```suggestion
		t.Run("user mode not enabled but user has public tag", func(t *testing.T) {
```

Or change other occurrences to use "disabled".